site/en/blog/framework-tools-font-fallback/index.md

```diff
@@ -0,0 +1,402 @@
+        }
+    ```
+
+{% Aside %}
+
+The next/font component is designed to support only ‘Arial’ and ‘Times New Roman’ as the fallback fonts. This choice was made mainly because of the universal support for Arial and Times New Roman fonts across platforms.
+
+{% endAside %}
+
+{% Aside %}
+
+The [adjustFontFallback](https://nextjs.org/docs/api-reference/next/font#adjustfontfallback) config option:
+
+For @next/font/google: A boolean value that sets whether an automatic fallback font should be used to reduce Cumulative Layout Shift. The default is true. Next.js automatically sets your fallback font to either ‘Arial’ or ‘Times New Roman’ depending on the font type (serif vs sans-serif respectively).
+
+For @next/font/local: A string or boolean false value that sets whether an automatic fallback font should be used to reduce Cumulative Layout Shift. The possible values are 'Arial', 'Times New Roman', or false. The default is `Arial`. If you want to use a serif font, consider setting this value to `Times New Roman`.
+
+{% endAside %}
+
+
+### Another option for Google fonts
+
+If using the next/font component isn’t an option, another approach to use this feature with Google Fonts is via the `optimizeFonts` flag. Next.js has the optimizeFonts feature already enabled by default. This feature inlines the Google Font CSS in the HTML response. Further, you can enable the font fallbacks adjustment feature by setting the `experimental.adjustFontFallbacksWithSizeAdjust` flag in your next.config.js, as shown in the following snippet:
+
+
+```javascript
+// In next.config.js
+module.exports = {
+ experimental: {
+   adjustFontFallbacksWithSizeAdjust: true,
+ },
+}
+```
+
+
+**Note**: There is no plan to support this feature with the app dir. In the long term, it’s ideal to use `next/font`.
+
+
+## How to adjust font fallbacks with Nuxt 
+
+[@nuxtjs/fontaine](https://github.com/nuxt-modules/fontaine) is a module for the Nuxt.js framework that automatically calculates the fallback font metric values and generates the fallback @font-face CSS.
+
+Enable the module by adding "@nuxtjs/fontaine" to your modules config:
+
+```javascript
+    import { defineNuxtConfig } from 'nuxt'
+
+    export default defineNuxtConfig({
+        modules: ['@nuxtjs/fontaine'],
+    })
+```
+
+{% Aside %}
+
+If you use Google Fonts or don't have a @font-face declaration for a font, you can declare them as additional options.
+
+  
+
+In most cases, the module can read the @font-face rules from your CSS and automatically infer the details such as the font-family, fallback font family, and display type.
+
+If the font is defined in a place not discoverable by the module, you can pass the metrics info as shown in the following code snippet.
+
+{% endAside %}
+
+```javascript
+    export default defineNuxtConfig({
+    modules: ['@nuxtjs/fontaine'],
+    fontMetrics: {
+     fonts: ['Inter', { family: 'Some Custom Font', src: '/path/to/custom/font.woff2' }],
+    },
+    })
+```
```

Comment:
```suggestion
```javascript
export default defineNuxtConfig({
  modules: ['@nuxtjs/fontaine'],
  fontMetrics: {
  fonts: ['Inter', { family: 'Some Custom Font', src: '/path/to/custom/font.woff2' }],
},
})
```
```